.github/workflows/scripts/get_compilation_flags.js

```diff
@@ -77,6 +64,29 @@ module.exports = async ({github, context, core}) => {
         };
     }
 
+    // Get the GitHub event name that triggered the workflow
+    const eventName = process.env.GITHUB_EVENT_NAME;
+    if (eventName == "schedule")
```

Comment:
Instead of doing this, is it possible to pass label as an input ?

One way to doing this is to pass an environment variable as input :
- https://github.com/actions/github-script?tab=readme-ov-file#use-env-as-input
- https://github.com/actions/github-script?tab=readme-ov-file#run-a-separate-file-with-an-async-function

If this environment variable is setup use it as labelNames. If the environment variable is not setup, we extract label from data.labels.